tslint.json

```diff
@@ -7,6 +7,9 @@
   ],
   "rules": {
     "prettier": true,
+    "deprecation": {
+      "severity": "warning"
+    },
```

Comment:
Which deprecation errors does this change to warnings?

Reply:
Not sure. It was added by the migration tool. Hard to find any information online.

Reply:
Not sure. We can try to change it back to `esnext` and see if it breaks.

Reply:
Hmm, I found this: https://palantir.github.io/tslint/rules/deprecation/

It's not recently updated (because TS Lint has been deprecated for 2 years 😊) but it _does_ say:

> Any usage of an identifier with the @deprecated JSDoc annotation will trigger a warning. See http://usejsdoc.org/tags-deprecated.html